src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -273,16 +316,21 @@ export function CollapsibleNav({
         {/* OpenSearchDashboards, Observability, Security, and Management sections */}
         {orderedCategories.map((categoryName) => {
           const category = categoryDictionary[categoryName]!;
+          const opensearchLinkLogo =
+            category.label === 'OpenSearch Dashboards'
```

Comment:
can this constant be pulled out a little higher like "DEFAULT_APP_TITLE" so it's easy to change.

Reply:
actually can we copy what we are doing here src/plugins/opensearch_dashboards_overview/public/application.tsx basing it off the id. then I dont see this value changing.

Reply:
I put opensearchLinklogo constant there because we need the category value for each item in the map, so i think it is more convenient to put the logic inside and after we get the category value. Thx!

Reply:
Yea i changed the logic so it bases off category.id.